packer_test/commands_test.go

```diff
@@ -5,15 +5,16 @@ import (
 	"os"
 	"os/exec"
 	"strings"
-	"sync"
+	"sync/atomic"
 	"testing"
 )
 
 type packerCommand struct {
-	once       sync.Once
+	runs       *atomic.Int32
```

Comment:
Does this need to be a pointer? 

The internal implementation uses pointer semantics to modify the counter and the usage within the test harness only modifies using the types API so making this a pointer doesn't seem right to me. But I've not worked with this type in the past so I can be wrong.

Reply:
Note: rerolled with a plain `int` instead of an atomic